### ISSUE TITLE
throw er; // Unhanded 'error' event

### ISSUE THREAD

**Issue by jmkoni** [notwaldorf/tiny-care-terminal#22]
I set up all my environment variables, updated npm (now using 4.5.0), and ran `npm install -g tiny-care-terminal`. However, when I run `tiny-care-terminal`, I get this:

```
   Error: spawn sh /usr/local/lib/node_modules/tiny-care-terminal/standup-helper.sh ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
    at Function.Module.runMain (module.js:469:11)
    at startup (node.js:134:18)
    at node.js:961:3
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn sh /usr/local/lib/node_modules/tiny-care-terminal/standup-helper.sh ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
```

Not super familiar with npm or node.

**Comment by DJWassink** [comment]
Seems like this is the same as issue #14 but im not entirely sure. But its something related with git-standup. By any change did you install `git-standup`?

See: https://github.com/notwaldorf/tiny-care-terminal#configure-the-dashboard

**Comment by DanielAndreasen** [comment]
Try run `npm install -g git-standup` first. Not sure if this will help, but it is needed if you don't have it already. There is already a PR #18 trying to improve this.

**Comment by jmkoni** [comment]
I did install `git-standup` and it seems to work great. Still can't get `tiny-care-terminal` to run. :(

**Comment by jmkoni** [comment]
One additional thing:
`DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`

That line does not return the directory that `node-modules` is in. However, even if I replace it with the directory that `node-modules` is in, I still get the same result.

**Comment by jmkoni** [comment]
And now, if I run `sudo sh ./standup-helper.sh ~/dev/` (with `~/dev/` being where my repos are), I get the results of `git-standup`.

**Comment by notwaldorf** [comment]
Duplicate of #14
There's a bunch of problems with `git-standup`; I'm just going to aggregate them under one issue

**Comment by notwaldorf** [comment]
@jmkoni yeah, that `DIR` line only returns the right node_modules directory for a particular install of npm, i think (not with yarn, maybe using nvm) :(

**Comment by notwaldorf** [comment]
Cross posted from #14:
OK! I published `1.0.4`  that _maybe_ works better? Anyone with a broken `tiny-care-terminal` want to install that and tell me if it's better, pretty please? 🙏
